README.rst

```diff
@@ -86,6 +86,7 @@ The minimum required dependencies to run MNE-Python are:
 - NumPy >= 1.20.2
 - SciPy >= 1.6.3
 - Matplotlib >= 3.4.0
+- NiBabel >= 3.2.1
```

Comment:
we discovered this in HNN. Looks like `nibabel` is a core dependency now but it's not reflected in the `setup.py`: https://github.com/mne-tools/mne-python/blob/main/requirements_base.txt

Reply:
Can you give a traceback? We ended up *not* making it a core dependency

Reply:
here you go: https://github.com/jonescompneurolab/hnn-core/pull/649#issuecomment-1734313692

should we be installing `nibabel` in our CIs separately?

Reply:
Yes, `nibabel` is an optional dependency not a core one, so you have to install it separately. PR welcome to change the wording of the error message to say something about it being optional and to consider installing it

https://github.com/mne-tools/mne-python/blob/079c868240a898204bf82b2f1bf0e04cdee75da1/mne/utils/check.py#L159